reference/supervisor.md

```diff
@@ -81,6 +81,7 @@ scored.
 
 WbNodeRef wb_supervisor_node_get_from_def(const char *def);
 WbNodeRef wb_supervisor_node_get_from_id(int id);
+const char *wb_supervisor_node_get_def(WbNodeRef node);
```

Comment:
I will move it one line up (just after `wb_supervisor_node_get_from_def`) instead of splitting `wb_supervisor_node_get_from_id` from `wb_supervisor_node_get_id`.

Same commet for line 75 and 118.